libpod/sqlite_state.go

```diff
@@ -373,10 +373,6 @@ func (s *SQLiteState) ValidateDBConfig(runtime *Runtime) (defErr error) {
 		return fmt.Errorf("retrieving DB config: %w", err)
 	}
 
-	if err := tx.Commit(); err != nil {
-		return fmt.Errorf("committing database validation row: %w", err)
-	}
```

Comment:
Maybe a stupid question, but in the interests of keeping the transaction as short as possible, might it make sense to instead keep this here, add `tx = nil`, and check for `tx != nil` in the `defer`?

Reply:
we can do that but the checks are just string compare so it is very fast anyway

Reply:
Thanks. LGTM as it is.